.gitignore

```diff
@@ -0,0 +1,2 @@
+# ignore maven target directory
+target/
```

Comment:
What about IntelliJ files and things like `.DS_STORE`?

Reply:
Yeah, good idea.

I have a global gitignore that ignores those, but probably worth putting them in the project gitignore.